app/dashboard/models.py

```diff
@@ -2871,6 +2871,8 @@ class Profile(SuperModel):
     objects_full = ProfileQuerySet.as_manager()
     brightid_uuid=models.UUIDField(default=uuid.uuid4, unique=True)
     is_brightid_verified=models.BooleanField(default=False)
+    is_twitter_verified=models.BooleanField(default=False)
+    twitter_handle=models.CharField(blank=True, max_length=15)
```

Comment:
could we set null=True as well ? else we won' be able to edit Profiles via admin if they don't have a value for twitter_handle

Reply:
Done!